package.json

```diff
@@ -72,7 +73,7 @@
         "semver": "^7.6.0"
     },
     "peerDependencies": {
-        "aws-cdk-lib": "2.133.0",
-        "aws-cdk": "2.133.0"
+        "aws-cdk": "2.133.0",
+        "aws-cdk-lib": "2.133.0"
```

Comment:
What changed?

Reply:
It looks like I just changed the placement of the aws-cdk above aws-cdk-lib. No real change and actually not necessary.